config/materials/aisi_630.mtl

```diff
@@ -0,0 +1,86 @@
+﻿ND_RelParSet_K01 = {
+
+Name = AISI_630
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_TYPE
+  Type = Integer
+  Default = 9
+  Access = Locked
+},
+{
+  Name = PTC_FAILURE_CRITERION_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_FATIGUE_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Stainless steel with excellent mech props. Aka 17-4PH.
+Src: matweb, sidex'
+  Access = Full
+},
+{
+  Name = TEMPERATURE
+  Type = Real
+  Default = 0.000000e+00 C
+  Access = Full
+},
+{
+  Name = PTC_INITIAL_BEND_Y_FACTOR
+  Type = Real
+  Default = 5.000000e-01
+  Access = Full
+},
+{
+  Name = PTC_BEND_TABLE
+  Type = String
+  Default = ''
+  Access = Full
+},
+{
+  Name = PTC_MASS_DENSITY
+  Type = Real
+  Default = 7.750000e+00 g/cm^3
+  Access = Full
+},
+{
+  Name = PTC_POISSON_RATIO
+  Type = Real
+  Default = 2.720000e-01
```

Comment:
@Lawproto as discussed F2F I would use the numbers in the material datasheet.

Reply:
@fiorisi ok, changes implemented in the new commit. According to the datasheet, cond. A falls between H 1100 and H 1150 in terms of mechanical properties so I did an average between the two, when the cond. A value was not available directly.

![immagine](https://user-images.githubusercontent.com/84333301/202503363-88a543e7-1ab5-4765-a2e2-d98d8344ec62.png)